aten/src/ATen/core/jit_type.h

```diff
@@ -435,6 +435,17 @@ struct TORCH_API SymbolicShape {
     return dims_;
   }
 
+  c10::optional<std::vector<bool>> concreteDims() const {
+    if (!dims_) {
+      return c10::nullopt;
```

Comment:
which cases is !dims_ true?  Is this just for ops that aren't yet supported, or it's also going to happen sometimes regardless of 'full op coverage'?

Reply:
There are certain ops that can create dynamic number of dims (eg `torch.squeeze`), so it can still rarely happen even when there is full op coverage. Our current plan is to ignore / fail on the dynamic number of dims case.

Reply:
hopefully we should be able to create rank specialized graphs to avoid this issue, but we will see when we get there.